### PR TITLE
Fix #91 and #88

### LINKED ISSUE #91: getPressedTicks()
getPressedTicks() Does not seem to exist?

### PR DESCRIPTION
**Pull request by IhorNehrutsa**
Add getPressedMs() function.

Add LongPressEvents.ino example.
```
OneButton Example.
Please press and hold the button for a few seconds.
810	 - LongPressStart()
820	 - DuringLongPress()
1820	 - DuringLongPress()
2820	 - DuringLongPress()
3820	 - DuringLongPress()
4820	 - DuringLongPress()
5820	 - DuringLongPress()
6550	 - LongPressStop()
```
In the first column, milliseconds from the start of pressing the button.

### CODE REVIEW

File: src/OneButton.h [hunk 1]
```diff
@@ -223,15 +230,24 @@ class OneButton
   unsigned long _lastDebounceTime = 0; // millis()
   unsigned long now = 0;               // millis()
 
-  unsigned long _startTime = 0; // start of current input change to checking debouncing
+  unsigned long _startTime = 0; // start time of current activeLevel change
   int _nClicks = 0;             // count the number of clicks with this variable
   int _maxClicks = 1;           // max number (1, 2, multi=3) of clicks of interest by registration of event functions.
 
+  unsigned int _long_press_intrval_ms = 0; // interval in msecs between calls of the DuringLongPress event
```
Comment (mathertel) [inline src/OneButton.h:237]: Please fix the typo: _long_press_intrval_ms => _long_press_interval_ms

**Comment by mathertel** [comment]
Thanks a lot for the contribution. This is welcome.

Please fix the typo: _long_press_intrval_ms => _long_press_interval_ms.
Please add some instructive comments at the begin of the new example.

**Review by mathertel — APPROVED** [review]

**Comment by mathertel** [comment]
Thanks a lot !

**Comment by IhorNehrutsa** [comment]
Please bump the version and library.json 
Thanks

**Pull request merged**